db/db.json

```diff
@@ -1,136 +1,159 @@
 {
+  "employers": [],
```

Comment:
thanks for putting the db.json.
this made it easier to test

Reply:
I excluded it first, because I did not want to accidentally push spuriously updated db.json files all the time. Then, when I realized that it would be good to push an updated one, I somehow couldn't (even when removing db.json from my gitignore list). Once Cami pushed her changes, db.json was in those, too, and all of a sudden I could push my changes. Pretty weird.